### PR TITLE
[fix][txn] Correct the prompt message

### PR DESCRIPTION
**Pull request by thetumbled**
### Motivation
the prompt message is wrong.

### Modifications
change the prompted message.

### CODE REVIEW

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/pendingack/impl/PendingAckHandleImpl.java [hunk 1]
```diff
@@ -181,7 +181,7 @@ private void addIndividualAcknowledgeMessageRequest(TxnID txnID,
     public void internalIndividualAcknowledgeMessage(TxnID txnID, List<MutablePair<PositionImpl, Integer>> positions,
                                                      CompletableFuture<Void> completableFuture) {
         if (txnID == null) {
-            completableFuture.completeExceptionally(new NotAllowedException("Positions can not be null."));
+            completableFuture.completeExceptionally(new NotAllowedException("txnID can not be null."));
```
Comment (RobertIndie) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/pendingack/impl/PendingAckHandleImpl.java:184]: Could you explain it in more detail? Why changed here?

**Comment by thetumbled** [comment]
The condition checks if txnID is null,not the&nbsp;Positions.isn't it?




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "apache/pulsar"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年8月9日(星期二) 晚上7:32
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [apache/pulsar] Correct the prompt message (PR #17009)





 
@RobertIndie commented on this pull request.
 
 
In pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/pendingack/impl/PendingAckHandleImpl.java:
 &gt; @@ -181,7 +181,7 @@ private void addIndividualAcknowledgeMessageRequest(TxnID txnID,      public void internalIndividualAcknowledgeMessage(TxnID txnID, List<MutablePair<PositionImpl, Integer&gt;&gt; positions,                                                       CompletableFuture<Void&gt; completableFuture) {          if (txnID == null) { -            completableFuture.completeExceptionally(new NotAllowedException("Positions can not be null.")); +            completableFuture.completeExceptionally(new NotAllowedException("txnID can not be null."));  
Could you explain it in more detail? Why changed here?

**Review by codelipenghui — APPROVED** [review]

**Review by Jason918 — APPROVED** [review]
LGTM

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/pendingack/impl/PendingAckHandleImpl.java (hunk 1 above)
Comment (thetumbled) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/pendingack/impl/PendingAckHandleImpl.java:184]: The condition checks if txnID is null, not the Positions, isn't it?

**Review by RobertIndie — APPROVED** [review]

**Comment by congbobo184** [comment]
@thetumbled  hi, I move this PR to release/2.9.5, if you have any questions, please ping me. thanks.

**Comment by thetumbled** [comment]
> @thetumbled hi, I move this PR to release/2.9.5, if you have any questions, please ping me. thanks.

Changes have been approved by reviewers. could you merge it? thanks.

**Comment by congbobo184** [comment]
@thetumbled hi, could you please merge the `apache/master` branch, seem the test can't rerun

**Comment by thetumbled** [comment]
> @thetumbled hi, could you please merge the `apache/master` branch, seem the test can't rerun

done.

**Review by nicoloboschi — APPROVED** [review]

**Pull request merged**